bigframes/ml/imported.py

```diff
@@ -32,15 +32,17 @@ class TensorFlowModel(base.Predictor):
     """Imported TensorFlow model.
 
     Args:
+        model_path (str):
+            GCS path that holds the model files.
         session (BigQuery Session):
             BQ session to create the model
-        model_path (str):
-            GCS path that holds the model files."""
+    """
 
     def __init__(
```

Comment:
Should we mark this breaking change? https://www.conventionalcommits.org/en/v1.0.0/#:~:text=in%20Semantic%20Versioning).-,BREAKING%20CHANGE,-%3A%20a%20commit

Reply:
It is marked as breaking, see the "!" in PR title.